unittest/build_order_test.cpp

```diff
@@ -56,20 +56,20 @@ ut::suite<"Build Order"> build_order_suite = [] {
 
         helper.parse_orders(faction->num, ss, nullptr);
 
-        expect(leader->object->region->objects.size() == 3_ul); // dummy + shaft + tower
+        expect(leader->object->region->objects.size() == 3); // dummy + shaft + tower
 
         helper.run_month_orders();
 
-        expect(leader->oldorders.empty() == "true"_b);
-        expect(unit->oldorders.empty() == "true"_b);
-        expect(leader->object->incomplete == 8_i);
+        expect(leader->oldorders.empty());
+        expect(unit->oldorders.empty());
+        expect(leader->object->incomplete == 8);
```

Comment:
the previous versions of these give better more english error messages.  Please revert this specific change.
With your change, it will just say 'failed [false]' or something similar.  Without it it would say something like 'false' != 'true' (or in the case of incomplete) it would say 'false' with your change instead of '5 != 8' which is what it would say with the original version.

The test harness uses those _i _b, etc in a nifty way to get nice textual assertion output.

Reply:
It is unclear why you opted to change these also.

Reply:
Since some of those changes are still there, I’m blocking on this still.